R/tm_g_bivariate.R

```diff
@@ -371,26 +370,33 @@ srv_g_bivariate <- function(id,
       selectors
     })
 
-    merged_data <- teal.transform::data_merge_srv(
+    anl_merged_input <- teal.transform::merge_expression_srv(
       selector_list = reactive_select_input,
-      datasets = datasets
+      datasets = data,
+      join_keys = attr(data, "join_keys")
     )
 
-    plot_r <- reactive({
+    anl_merged_q <- reactive({
+      teal.code::new_quosure(env = data) %>%
+        teal.code::eval_code(as.expression(anl_merged_input()$expr))
+    })
+
+    merged <- list(
+      anl_input_r = anl_merged_input,
+      anl_q_r = anl_merged_q
+    )
```

Comment:
### Explanation comment
- merge expression needs to be evaluated in the quosure 
- and we join informations from anl_merged_input and anl_merged_q into one list to highlight that they are related and should be used together